.github/workflows/test.yml

```diff
@@ -11,9 +11,11 @@ jobs:
   tests:
     uses: ljharb/actions/.github/workflows/node.yml@main
     with:
-      range: '>= 0.4'
+      range: '>= 0.8'
       type: majors
-      command: npm run tests-only
+      build-command: npm run build:tests
+      build-output-dir: __tests-built__,lib
```

Comment:
Building test-only libs into `lib` seems like bad and potentially dangerous design to me. Better to use something like `lib-test`. Dont want future changes to lead to chained attacks like runtime depending on `lib` and tests modifying them.